Mathlib/Algebra/SkewMonoidAlgebra/Basic.lean

```diff
@@ -160,6 +153,269 @@ theorem support_eq_empty {p} : p.support = ∅ ↔ (p : SkewMonoidAlgebra k G) =
 
 end Support
 
+section Coeff
+
+/-- `coeff f a` (often denoted `f.coeff a`) is the coefficient of `a` in `f`. -/
+def coeff : SkewMonoidAlgebra k G → G → k
+  | ⟨p⟩ => p
+
+@[simp]
+theorem coeff_ofFinsupp (p) : coeff (⟨p⟩ : SkewMonoidAlgebra k G) = p := rfl
+
+theorem coeff_injective : Injective (coeff : SkewMonoidAlgebra k G → G → k) := by
+  rintro ⟨p⟩ ⟨q⟩
+  simp only [coeff, DFunLike.coe_fn_eq, imp_self, ofFinsupp.injEq]
+
+@[simp]
+theorem coeff_inj (p q : SkewMonoidAlgebra k G) : p.coeff = q.coeff ↔ p = q :=
+  coeff_injective.eq_iff
+
+theorem toFinsupp_apply (f : SkewMonoidAlgebra k G) (g) : f.toFinsupp g = f.coeff g := rfl
```

Comment:
```suggestion
@[simp]
theorem toFinsupp_apply (f : SkewMonoidAlgebra k G) (g) : f.toFinsupp g = f.coeff g := rfl
```
since `coeff` is the preferred spelling when working with `SkewMonoidAlgebra`